conf/modules/mag_hmc58xx.xml

```diff
@@ -44,6 +46,10 @@
     <configure name="MAG_HMC58XX_I2C_DEV" default="TRUE" case="upper|lower"/>
     <define name="USE_$(MAG_HMC58XX_I2C_DEV_UPPER)"/>
     <define name="MAG_HMC58XX_I2C_DEV" value="$(MAG_HMC58XX_I2C_DEV_LOWER)"/>
+    <configure name="MAG_HMC58XX_SA" value="0"/>
+    <configure name="MAG_HMC58XX_TC" value="0"/>
```

Comment:
Change the `default` so it can be configured by hand in airframe file (and if not recommended, remove from `doc`)
```suggestion
    <configure name="MAG_HMC58XX_SA" default="0"/>
    <configure name="MAG_HMC58XX_TC" default="0"/>
```